packages/rke2-canal/charts/templates/config.yaml

```diff
@@ -60,7 +60,7 @@ data:
   # Flannel network configuration. Mounted into the flannel container.
   net-conf.json: |
     {
-      "Network": {{ .Values.podCidr | quote }},
+      "Network": {{ .coalesce .Values.global.clusterCIDR Values.podCidr | quote }},
```

Comment:
Is this needed? I thought canal was using the kubernetes nodes CIDR as IPAM?

Reply:
I have just realized that Values.podCidr has a default value of "10.42.0.0/16". I guess using the kubernetes nodes CIDR as IPAM only happens if this value wouldn't exist, that's why this is needed, right?

Reply:
Yes. I thought the same thing, but @ibuildthecloud and @Oats87 clarified that this does need to be kept in sync with the cluster setting.